master/reference/calicoctl/commands/get.md

```diff
@@ -48,6 +48,9 @@ Options:
                                Uses the default namespace if not specified.
   -a --all-namespaces          If present, list the requested object(s) across 
                                all namespaces.
+  --export                     If present, returns the requested object(s) stripped of
```

Comment:
also need to add this around line 120

Reply:
Oops missed that. I'll add it in the options.